realm/realm-library/src/main/cpp/io_realm_SyncSession.cpp

```diff
@@ -28,22 +29,21 @@
 #include "jni_util/java_local_ref.hpp"
 #include "jni_util/jni_utils.hpp"
 
-using namespace std;
 using namespace realm;
 using namespace jni_util;
 using namespace sync;
 
 JNIEXPORT jboolean JNICALL Java_io_realm_SyncSession_nativeRefreshAccessToken(JNIEnv* env, jclass,
-                                                                              jstring localRealmPath,
-                                                                              jstring accessToken,
+                                                                              jstring j_local_realm_path,
```

Comment:
It stands for `java`. It is mostly a work-around so I don't have to different names for the same thing when the first I do is call `JStringAccessor`.

If it is confusing I guess I can go back to finding somewhat less sensible names for the method params.